jetty-runtime/jetty9-base/pom.xml

```diff
@@ -122,7 +135,7 @@
           <arguments>
             <argument>-jar</argument>
             <argument>../jetty-distribution-${jetty.version}/start.jar</argument>
-            <argument>--add-to-startd=gae,http,jsp,jstl</argument>
+            <argument>--add-to-startd=http,deploy,jsp,jstl,http-forwarded,resources,gae</argument>
```

Comment:
'deploy' adds the automatic discovery and deployment of the various webapp types from the `${jetty.base}/webapps` directory (this location has likely been configured to some other location on these images)

`http-forwarded` adds the proxy forward request customizers for incoming requests

`resources` adds the directory `${jetty.base}/resources` to the server classpath, so that libraries that look for configuration files via `Classloader.getResource()` can find their user provided files.  This is very popular for logging configuration files especially.  But can also be used for similar configuration purposes in other libraries.